webauthn-rs-core/src/attestation.rs

```diff
@@ -134,11 +214,7 @@ pub(crate) fn verify_packed_attestation(
             // (id-fido-gen-ce-aaguid) verify that the value of this extension matches the aaguid
             // in authenticatorData.
 
-            if let Some(aaguid) = get_fido_gen_ce_aaguid(attestn_cert) {
-                if acd.aaguid != aaguid {
-                    return Err(WebauthnError::AttestationCertificateAAGUIDMismatch);
-                }
-            }
+            validate_extension::<FidoGenCeAaguid>(&attestn_cert, &acd.aaguid)?;
```

Comment:
Look for `id-fido-gen-ce-aaguid` in the tpm attestation, because under assert_tpm_attest_req, it's also used there.